core/domain/services/admin/registrations/list_table/csv_reports/Checkins.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace EventEspresso\core\domain\services\admin\registrations\list_table\csv_reports;
+
+use EE_Checkin;
+use EE_Datetime;
+use EE_Error;
+use ReflectionException;
+
+/**
+ * Class Checkins
+ * Renders the Checkin data
+ *
+ * @author  Hossein Rafiei
+ * @package EventEspresso\core\domain\services\admin\registrations\list_table\csv_reports
+ */
+class Checkins
+{
+
+    /**
+     * Returns datetime label
+     *
+     * @param EE_Datetime $datetime
+     * @return string
+     * @throws EE_Error
+     * @throws ReflectionException
+     */
+    public static function getDatetineLabel(EE_Datetime $datetime)
+    {
+        if (trim($datetime->get('DTT_name'))) {
+            /* translators: 1: datetime name, 2: datetime ID */
+            $name = sprintf(
+                esc_html__('Check-ins for %1$s - ID: %2$s', 'event_espresso'),
+                esc_html($datetime->get('DTT_name')),
+                esc_html($datetime->get('DTT_ID'))
+            );
+        } else {
+            /* translators: %s: datetime ID */
+            $name = sprintf(
+                esc_html__('ID: %1$s', 'event_espresso'),
+                esc_html($datetime->get('DTT_ID'))
+            );
+        }
+
+        return $name;
```

Comment:
this is great, but it's even cleaner and easier to read if you remove the `$name` variable and just return immediately. 
this also allows you to get rid of the else clause which reduces complexity:

```php
        if (trim($datetime->get('DTT_name'))) {
            /* translators: 1: datetime name, 2: datetime ID */
            return sprintf(
                esc_html__('Check-ins for %1$s - ID: %2$s', 'event_espresso'),
                esc_html($datetime->get('DTT_name')),
                esc_html($datetime->get('DTT_ID'))
            );
        } 
        /* translators: %s: datetime ID */
        return sprintf(
            esc_html__('ID: %1$s', 'event_espresso'),
            esc_html($datetime->get('DTT_ID'))
        );
```

Reply:
Yes, It's better. Done.